src/hex/unittest/notifier/junit/JUnitTestNotifier.hx

```diff
@@ -71,7 +72,7 @@ class JUnitTestNotifier implements ITestRunnerListener
 		var summary = this._testSuitesInExecution.first();
 		updateSummary(summary, event);
 		summary.output += getTestCaseStart(event);
-		summary.output += "<failure type=\"" + event.getError().name + "\" message=\"" + event.getError().message + "\">" + event.getError() + "</failure>";
+		summary.output += "<failure type=\"" + event.getError().name + "\" message=\"" + event.getError().message.htmlEscape(true) + "\"><![CDATA[" + event.getError() + "]]></failure>";
 		summary.output += getTestCaseEnd();
```

Comment:
Even by using CDATA here there are small chances of things breaking if the message happens to have ]]>. So ]]> should be replaced with ]]]]><![CDATA[>.